.github/workflows/staging_pipeline.yml

```diff
@@ -230,3 +230,15 @@ jobs:
           cf set-env $APP_NAME CSV_DOWNLOAD_PAAS_INSTANCE $CSV_DOWNLOAD_PAAS_INSTANCE
           cf set-env $APP_NAME SENTRY_DSN $SENTRY_DSN
           cf push $APP_NAME --strategy rolling -t 180
+
+  aws_deploy:
+    name: AWS Deploy
+    if: github.ref == 'refs/heads/main'
+    needs: [lint, test, feature_test, audit]
```

Comment:
Do we need lint, test etc for this job? Is that not only necessary for the (non-AWS) deploy job?

Reply:
Yes - we want this deployment to happen if and only if the paas one does (ignoring failures during deployment, but the same triggers/prerequisites). If nothing else, it would be a very easy thing to miss when removing the paas deployment later.

Reply:
Gotcha, thanks!